syntax = "proto3";

message User {
  string user_name = 1;
  int32 age = 2;
  string email = 3;
}

service UserExampleService {
  rpc CreateUser(User) returns (User) {}
  rpc GetUser(User) returns (User) {}
  rpc UpdateUser(User) returns (User) {}
  rpc DeleteUser(User) returns (User) {}
}